docs/source/model-registry.rst

```diff
@@ -78,47 +79,170 @@ version to one of the other valid stages.
 
 .. code-block:: py
 
-   mlflow.tensorflow.log_model(tf_saved_model_dir=saved_estimator_path,
-                               tf_meta_graph_tags=[tag_constants.SERVING],
-                               tf_signature_def_key="predict",
-                               artifact_path="model",
-                               registered_model_name="EmailMarketingCampaign")
+   client = MlflowClient()
+   client.create_registered_model("sk-learn-random-forest-reg-model")
 
-.. note:: If the registered model does not already exist, the registry will create one before creating a new model version.
+While the method above creates an empty registered model with no version associated, the method below creates a new version of the model.
 
-To confirm that the model has successfully been registered, call the ``get_model_version_details`` API:
+.. code-block:: py
+
+    client = MlflowClient()
+    result = client.create_model_version(name="sk-learn-random-forest-reg-model",
+                source = "mlruns/0/d16076a3ec534311817565e6527539c0/artifacts/sk-model",
+                run_id = "d16076a3ec534311817565e6527539c0")
+
+In contrast, ``mlflow.register_model()`` and ``mlflow.model_flavor.log_model()`` will create a new version in the registry.
+
+Adding or Updating Model Descriptions
+-------------------------------------
+
+At any point in a model’s lifecycle development, you can update a model version's description using the MLflow Tracking API.
 
 .. code-block:: py
 
-   client = mlflow.tracking.MlflowClient()
-   client.get_model_version_details(name = "EmailMarketingCampaign",
-                                    version = 12)
+    client = MlflowClient()
+    client.update_model_version(name=”sk-learn-random-forest-reg-model”,
+            version=1,
+            description="This model version is a random forest containing 100 decision trees that was trained in scikit-learn.")
+
+As well as adding or updating a description of a specific version of the model, you can rename an existing registered model.
+
+.. code-block:: py
+
+    client=MlflowClient()
+    client.rename_registered_model(name=”sk-learn-random-forest-reg-model”,
+            new_name= ”sk-learn-random-forest-reg-model-100”)
+
+Transitioning an MLflow Model’s Stage
+-------------------------------------
+Over the course of the model’s lifecycle, a model evolves—from development to staging to production.
+You can transition a registered model in the registry to one of the stages: **Staging**, **Production** or **Archived**.
+
+.. code-block:: py
+
+    client = MlflowClient()
+    client.transition_model_version_stage(name=”sk-learn-random-forest-reg-model”,
+  	        version=3,
+	        stage=“production”)
```

Comment:
We should probably include an example of `archive_existing_versions`, right @mparkhe ?

Reply:
I believe this is only an edge feature. We don't expose that in the OSS.

Reply:
`archive_existing_versions` is not support in SQL backend